### PR TITLE
Fix broken link and nit layout

### PR DESCRIPTION
**Pull request by camilamacedo86**
Fix broken link and nit layout

### CODE REVIEW

File: website/content/en/docs/best-practices/multi-tenancy.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ and further information.
 [olm-docs]: /docs/olm-integration/
 [ocp-route]: https://docs.openshift.com/container-platform/4.9/rest_api/network_apis/route-route-openshift-io-v1.html#route-route-openshift-io-v1
 [ocp-ingress-doc]: https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html#nw-ingress-sharding-route-labels_configuring-ingress
+[k8s-ingress-controllers]: https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/
```
Comment (varshaprasad96) [inline website/content/en/docs/best-practices/multi-tenancy.md:53]: @camilamacedo86 Just wanted to check where is `k8s-ingress-controllers` being referenced? I couldn't find it in the doc, maybe its me missing it!

Comment (camilamacedo86) [reply website/content/en/docs/best-practices/multi-tenancy.md:53]: See: https://github.com/operator-framework/operator-sdk/blob/699f27b052f7fe6dbcd05e343d0ecd5b5c8c6f60/website/content/en/docs/best-practices/multi-tenancy.md?plain=1#L27-L35

Comment (camilamacedo86) [reply website/content/en/docs/best-practices/multi-tenancy.md:53]: Screenshot 2021-12-12 at 13 27 50

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Review by rashmigottipati — COMMENTED** [review]
/lgtm

**Pull request merged**